examples/react/basic/buf.gen.yaml

```diff
@@ -9,8 +9,14 @@ plugins:
       - target=ts
 
   - name: connect-query
-    path: ./node_modules/.bin/protoc-gen-connect-query
+    path: ./node_modules/.bin/protoc-gen-connect-query-react
     out: src/gen
     opt:
       - target=ts
       - import_extension=
+  - name: connect-query
+    path: ./node_modules/.bin/protoc-gen-connect-query-react
```

Comment:
Did you mean to duplicate these in this file?

Reply:
Yeah, i want to generate both d.ts files as well as .ts files, as a means of detecting the differences. Though in retrospect, perhaps I can put that elsewhere so it doesn't pollute the example 🤔

Reply:
Yeah our previous approach on examples in connect-es, et. al. has been to make the examples as straight-forward and understandable as possible. Just a thought though.